actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -173,11 +173,14 @@ def _back_url # :nodoc:
       #   link_to "Visit Other Site", "http://www.rubyonrails.org/", data: { confirm: "Are you sure?" }
       #   # => <a href="http://www.rubyonrails.org/" data-confirm="Are you sure?">Visit Other Site</a>
       #
-      # Also you can set target attribute with <tt>target</tt> options:
+      # Also you can set any link attributes such as <tt>target</tt>, <tt>rel</tt>, <tt>type</tt>, etc
+      # using html option with the same name:
```

Comment:
We're already stating they're attributes, so I don't think we need to repeat that. "Like `html_options`" also sounds off to me.

Thinking about this some more stopping it at "etc.:" is enough to me.